docs/android-frontend-testing.md

```diff
@@ -0,0 +1,143 @@
+          }
+        },
+        {
+          "slug": "control",
+          "ratio": 60,
+          "feature": {
+            "value": {},
+            "enabled": true,
+            "featureId": "the-feature-id"
+          }
+        }
+      ],
+      "outcomes": [],
+      "arguments": {},
+      "probeSets": [],
+      "startDate": "2021-04-19T22:40:44.614622Z",
+      "targeting": "true",
+      "featureIds": [
+        "the-feature-id"
+      ],
+      "application": "org.mozilla.ios.FirefoxBeta",
+      "bucketConfig": {
+        "count": 8000,
+        "start": 0,
+        "total": 10000,
+        "namespace": "nimbus-aa-validation-for-ios-v2-1",
+        "randomizationUnit": "nimbus_id"
+      },
+      "schemaVersion": "1.4.0",
+      "userFacingName": "A replica of the experiment under test",
+      "referenceBranch": "control",
+      "proposedDuration": 28,
+      "isEnrollmentPaused": false,
+      "proposedEnrollment": 7,
+      "userFacingDescription": "Is Nimbus working? This experiment tries to find out.",
+      "id": "nimbus-aa-validation-for-ios-v2",
+      "last_modified": 1619530368808
+    }
+  ]
+}
+```
+
+## The annotated guide to the experiment definition
+
+Many of the fields in the above JSON correspond to the experiment UI.
+
+These fields affect which OS, app and build the experiment is for. These should align with the developer or nightly builds of the app your using, on the platform you're using. If these don't line up, the app will not pick up this experiment.
+
+ * `appName`: pick `fenix` or `firefox_ios`
+ * `appId`: The experimenter UI should help you select the right one.
+ * `channel`: this must be `nightly` or `developer`. The experimenter UI should help you select the right one.
+ * `feature_id`/`featureIds`: this is the identifier of the app feature under test. This should match what is hard coded into the App. Experimenter will put these in all the right places. If the feature id is 
+
+These fields affect segment the population for eligibility for the experiment, and which branch they'll be given.
+
+ * `targeting`: Experimenter will help you generate this JEXL query string. If Nimbus evaluates this on a given device to `true`, then the device is eligible for the experiment. If your testing the app, then you 
+ * `bucketConfig`: `start` `count` and `total`. Of the total eligible users, the proportion that will actually be enrolled in the experiment is given by `(count - start) / total`. For testing purposes, you should make `start = 0`, `count = 100000`, `total = 100000`, i.e. enroll 100% of eligible devices.
+
+If the device is enrolled in the experiment (i.e. is targeted as eligible, and in the experiment bucket), then it will be enrolled in to one of the two or more branches. Once enrolled it will not change branches.
+
+ * `branch` -> `slug`: This should match what the branches that the app's feature responds to. In most cases, it will be `treatment` and `control`.
```

Comment:
```suggestion
 * `branch` -> `slug`: This should match the branches that the app's feature responds to. In most cases, it will be `treatment` and `control`.
```